src/status_im2/contexts/chat/photo_selector/view.cljs

```diff
@@ -6,28 +6,27 @@
             [quo2.foundations.colors :as colors]
             [react-native.core :as rn]
             [react-native.linear-gradient :as linear-gradient]
-            [reagent.core :as reagent]
             [status-im2.contexts.chat.photo-selector.style :as style]
             [status-im.utils.core :as utils]
             [quo.react]
             [utils.re-frame :as rf]))
 
-(def selected (reagent/atom []))
-
 (defn on-press-confirm-selection
   [chat-id]
-  (rf/dispatch [:chat.ui/clear-sending-images chat-id])
-  (doseq [item @selected]
-    (rf/dispatch [:chat.ui/camera-roll-pick item]))
-  (reset! selected [])
-  (rf/dispatch [:bottom-sheet/hide]))
+  (let [selected (rf/sub [:chats/selected-photos])]
```

Comment:
better to do not use subs in functions, better to have them in view and pass as parameter